TerminalDocs/customize-settings/rendering.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Windows Terminal Rendering Settings
+description: Learn how to customize rendering settings within Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 1/21/2021
+ms.topic: how-to
+ms.localizationpriority: high
+---
+
+# Rendering settings in Windows Terminal
+
+The properties listed below affect the entire terminal window, regardless of the profile settings. These should be placed at the root of your settings.json file.
+
+If you are thinking about changing the rendering settings, additional information is provided on the [Troubleshooting page](./../troubleshooting.md#the-text-is-blurry) to help guide you.
+
+> [!TIP]
+> If you'd like to edit these settings using the settings UI, you will have to add the `"openSettings"` action to your `"actions"` array in order to open it with the command palette or keyboard.
```

Comment:
Link [this](https://review.docs.microsoft.com/en-us/windows/terminal/troubleshooting?branch=release-1.6#open-the-settings-ui) in the tip?

Reply:
Mildly concerned that we're saying "add the 'openSettings' action", because `{ "command": "openSettings" }` is valid, but that just adds the settings.json action, not SUI action.

Oh wait, you literally put the code block below. Any way that code block can be inside the tip? Feels a bit weird. If not, I say just link the relevant troubleshooting page secion and cut the code block.